styles.css

```diff
@@ -0,0 +1,53 @@
+body {
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Helvetica Neue',
+    Arial,
+    sans-serif;
+}
+
+main {
+  background-color: #eee;
+}
+
+#title,
+p {
+  text-align: center;
+  padding-top: 20px;
+}
+
+#image {
+  display: block;
+  max-width: 100%;
+  height: auto;
+  margin-left: auto;
+  margin-right: auto;
+  justify-content: center;
+}
+
+#img-caption {
+  text-align: center;
+}
+
+#tribute-info {
+  display: block;
+  width: 70%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+#tribute-info ul li {
```

Comment:
I think you should simply use li instead of #tribute-info ul li Selector; Because it will do the same thing as above,
Document is not very big. So, keep it Simple 💯 🧠

Reply:
> I think you should simply use li instead of #tribute-info ul li Selector; Because it will do the same thing as above, Document is not very big. So, keep it Simple 💯 🧠

Please consider not using the words like `should or ought or something else` as you are not supposed to give orders else you are here to advise the user of a descriptive change with a valid reason for the change. Consider editing your comment to make it in accordance with the above mentioned instructions 👍